cmake/options.cmake

```diff
@@ -200,6 +200,10 @@ macro(drake_setup_options)
     DEPENDS "HAVE_BOT_CORE_LCMTYPES"
     "robotlocomotion LCM types")
 
+  drake_system_dependency(VTK REQUIRES VTK VERSION 5.8
```

Comment:
Hmm... alas, VTK does appear to make it strict. Do we have 5.10 everywhere? If so, we can just change it, otherwise I guess we need to drop the version check. (That's unfortunate, as we would like to disallow VTK 6+, but the problem is on the VTK side.)